src/none-driver.ts

```diff
@@ -2,34 +2,46 @@ import {getInput} from '@actions/core'
 import {exec} from '@actions/exec'
 import {downloadTool} from '@actions/tool-cache'
 
-const installCriDocker = async (): Promise<void> => {
-  const urlBase =
-    'https://storage.googleapis.com/setup-minikube/cri-dockerd/v0.2.3/'
-  const binaryDownload = downloadTool(urlBase + 'cri-dockerd')
-  const serviceDownload = downloadTool(urlBase + 'cri-docker.service')
-  const socketDownload = downloadTool(urlBase + 'cri-docker.socket')
-  await exec('chmod', ['+x', await binaryDownload])
-  await exec('sudo', ['mv', await binaryDownload, '/usr/bin/cri-dockerd'])
-  await exec('sudo', [
-    'mv',
-    await serviceDownload,
-    '/usr/lib/systemd/system/cri-docker.service',
-  ])
+const installCniPlugins = async (): Promise<void> => {
+  const cniPluginsURL =
+    'https://github.com/containernetworking/plugins/releases/download/v1.2.0/cni-plugins-linux-amd64-v1.2.0.tgz'
```

Comment:
As evident by the old versions in this file, it's easy to forget to update these, I can add automation to create PRs to update these. To assist with this could you make a variable for the version of CNI plugins to assist with this automation. Just something like `const cniPluginsVersion = "v1.2.0"`

Reply:
Good suggestion, Renovate's regex manager would be my go-to for such task: https://docs.renovatebot.com/modules/manager/regex/